### PR TITLE
Add .gitignore

### PR DESCRIPTION
**Pull request by cnasikas**
Add .gitignore file to ignore node_modules and package-lock.json

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```
Comment (dionyziz) [inline .gitignore]: [According to documentation](https://docs.npmjs.com/files/package-lock.json), package-lock.json "is intended to be committed into source repositories, and therefore should not be git ignored".

**Review by dionyziz — CHANGES_REQUESTED** [review]

**Comment by cnasikas** [comment]
I made a mistake and this commits are included in pull request #2 so I am closing it.

**Review by dionyziz — CHANGES_REQUESTED** [review]
This PR implies that we will use npm for this project. However, the latest recommendation is yarn. In that direction, perhaps we can use a [`yarn.lock`](https://yarnpkg.com/lang/en/docs/yarn-lock/) file instead. What do you think?

**Comment by cnasikas** [comment]
Of course. But we already have a yarn.lock file, right? According to docs we should commit it. We could support both npm and yarn.

**Pull request closed without merging**